data.sql

```diff
@@ -1,32 +1,442 @@
+('Bob', 45 ),('Melody Pond', 77 ),('Dean Winchester', 14 ),('Jodie Whittaker', 38 );
+
+-- insert data for species
+
+INSERT INTO  species(name) VALUES('Pokemon'), ('Digimon');
+
+-- Modify your inserted animals so it includes the species_id value
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') 
+
+WHERE name LIKE '%mon';
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') 
+WHERE name NOT LIKE '%mon';
+
+-- Modify your inserted animals to include owner information (owner_id)
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Sam Smith'
+    )
+WHERE name = 'Agumon';
+UPDATE 1
+
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Jennifer Orwell'
+    )
+WHERE name = 'Gabumon';
+UPDATE 1
+
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Bob'
+    )
+WHERE name = 'Plantmon' AND 'Devimon';
+
+
+ UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Bob'
+    )
+                              ^
+UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Bob'
+    )
+WHERE name = 'Plantmon' OR name= 'Devimon';
+
+UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Melody Pond'
+    )
+WHERE name = 'Charmander' OR name= 'Squirtle' OR name = 'Blossom';
+
+ UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Dean Winchester'
+    )
+WHERE name = 'Angemon' OR name= 'Boarmon';
+
+UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Jennifer Orwell'
+    )
+WHERE name = 'Pikachu';
+UPDATE animals
+SET owner_id = (
+        SELECT id
+        FROM owners
+        WHERE full_name = 'Sam Smith '
+    )
+WHERE name = 'Agumon';
+SELECT * FROM animals;
+
+-- insert data into vets table
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('Vet William Tatcher', 45, 'Apr 23 2000'),
+('Vet Maisy Smith', 26 ,'Jan 17 2019'),
+('Vet Stephanie Mendez', 64, 'May 04 1981'),
+('Vet Jack Harkness', 38,'Jun 08 2008');
+
+-- insert data into specializations
+
+INSERT INTO specializations(species_id, vets_id)
+VALUES ( (
+            SELECT id
+            FROM species
+            WHERE name = 'Pokemon'
+        ),
+        (
+            SELECT id
+            FROM vets
+            WHERE name = 'William Tatcher'
+        )
+        ),
+
+        (
+          (
+  SELECT id
+  FROM species
+  WHERE name = 'Digimon'
+    OR name = 'Pokemon'
+),
+(
+  SELECT id
+  FROM vets
+  WHERE name = 'Vet Stephanie Mendez'
+)
+),
+
+( (
+            SELECT id
+            FROM species
+            WHERE name = 'Digimon'
+        ),
+        (
+            SELECT id
+            FROM vets
+            WHERE name = 'Vet Jack Harkness'
+        )
+        );
+
```

Comment:
- [ ] Kindly don't forget to modify the 'Vet Stephanie Mendez' and 'Vet Stephanie Mendez' names after removing the `vet`. Your code might look like this:

```sql
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
```